Tools/WebKitTestRunner/TestInvocation.cpp

```diff
@@ -157,7 +157,7 @@ void TestInvocation::loadTestInCrossOriginIframe()
         "<script>"
         "    testRunner.dumpChildFramesAsText()"
         "</script>"
-        "<iframe src=\""_s, m_urlString.utf8().span(), "\" style=\"position:absolute; top:0; left:0; width:100%; height:100%; border:0\">"_s));
+        "<iframe src=\""_s, m_urlString.utf8().span(), "?inCrossOriginFrame=true\" style=\"position:absolute; top:0; left:0; width:100%; height:100%; border:0\">"_s));
```

Comment:
Note there are tests loaded which already have query strings and/or fragments, where this won't work.